lib/tiled/attributes.rb

```diff
@@ -11,6 +11,9 @@ def initialize(hash)
           def #{key}
             @#{key}
           end
+          def #{key}=(value)
+            @#{key} = value
+          end
```

Comment:
These setters are added to support [this attribute manipulation](https://github.com/wildfiler/drtiled/pull/15/commits/a0f049eb0b820b93a0af5d17c75533e565c5db1a#diff-9c461043ca8f784cd56c48b6d390e6180a163415f14f8917a4e1d415606f1272R25-R28). Looking back on this, it seems kind of ugly and I will try to work around it.

Reply:
Addressed in force push [1be1ef4](https://github.com/wildfiler/drtiled/pull/15/commits/1be1ef4301466d4f788beb00473d2de92bfab8bb).